src/renderers/WebGLRenderer.d.ts

```diff
@@ -307,21 +307,14 @@ export class WebGLRenderer implements Renderer {
 	resetGLState(): void;
 	dispose(): void;
 
-	/**
-	 * Tells the shadow map plugin to update using the passed scene and camera parameters.
-	 *
-	 * @param scene an instance of Scene
-	 * @param camera — an instance of Camera
-	 */
 	renderBufferImmediate(
 		object: Object3D,
-		program: Object,
-		material: Material
+		program: WebGLProgram,
```

Comment:
Isn't it necessary to import `WebGLProgram`?

Reply:
You are correct. On the other hand it is no longer necessary to import `Fog`.

Reply:
Um, I wonder why Travis did not complain 🤔

Reply:
My IDE did not complain as well. Perhaps they are not taking the d.ts that seriously. No idea why.

Reply:
Ah, it's because `WebGLProgram` is actually a native type.